### ISSUE TITLE
Email is always nil

### ISSUE THREAD

**Issue by TFarla** [ueberauth/ueberauth_github#9]
When trying to get the email within the controller callback. Email seems to always be nil.
```
def callback(%{assigns: %{ueberauth_auth: auth}} = conn, _params, _current_user, _claims) do
  # auth.info.email = nil
end
```
The config:
```
config :ueberauth, Ueberauth,
  providers: [github: {Ueberauth.Strategy.Github, [default_scope: "user"]}]

config :ueberauth, Ueberauth.Strategy.Github.OAuth,
  client_id: System.get_env("GITHUB_CLIENT_ID"),
  client_secret: System.get_env("GITHUB_CLIENT_SECRET")
```

Is there anything I'm doing wrong here? I do request the permission to read the user's email so it should not be nil, right? Other information like "name" is being fetched correctly.

**Comment by hassox** [comment]
Hey @TFarla. I've used the 'login' scope previously and got the email. If you add that do you see it?

**Comment by TFarla** [comment]
@hassox I tried using the 'login' scope. Unfortunately still had the same error 
```
nil given for :email, comparison with nil is forbidden as it always evaluates to false. Pass a full query expression and use is_nil/1 instead.
```

EDIT: After trying the github login in [the phoenix_gaurdian repo](https://github.com/hassox/phoenix_guardian) and still getting the same error. I figured that the problem had to be somewhere in the settings of my profile. Turns out my public email was invisible.
![github settings page showing email visibility](http://puu.sh/mVgUg/e7700e1a78.png).

I'm closing this issue now since this is a mild case of PEBCAK.
